draft-packages/tile/KaizenDraft/Tile/__snapshots__/Tile.spec.tsx.snap

```diff
@@ -177,10 +177,10 @@ exports[`<InformationTile /> snapshots renders InformationTile with information
         >
           <button
             aria-disabled="true"
-            aria-label="Information"
+            aria-label="Back"
             class="button iconButton"
             disabled=""
-            title="Information"
+            title="Back"
```

Comment:
"Back" feels a bit anaemic here (though better than "Information" which never really made sense), wondering if @dougmacknz has thoughts on how we could better communicate to screen readers what’s happening here.

Reply:
I don't think there's any way to have this interaction be good for screen readers. The best solution I see here is that we hide the fact that there's an interaction at all, and just append the extra information to the DOM as _visually hidden_.

That would remove the need for this label at all, which means we don't have to worry about localising it as well.

Reply:
That seems fair, though perhaps outside the scope of @dabarrell’s PR. Reckon an `#accessibility` tagged issue for someone to follow-up could be a way to go, @dougmacknz?

Reply:
Created https://github.com/cultureamp/kaizen-design-system/issues/1764.

I would keep this as "Information" for now @dabarrell, "Back" makes less sense I would say.

Reply:
I've reverted this :)